config/clusterapi/bootstrap/manager/manager.yaml

```diff
@@ -0,0 +1,76 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    control-plane: controller-manager
+    app.kubernetes.io/name: namespace
+    app.kubernetes.io/instance: k0smotron
+    app.kubernetes.io/component: manager
+    app.kubernetes.io/created-by: k0smotron
+    app.kubernetes.io/part-of: k0smotron
+    app.kubernetes.io/managed-by: kustomize
+  name: k0smotron
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: controller-manager
+  namespace: k0smotron
+  labels:
+    control-plane: controller-manager
+    app.kubernetes.io/name: deployment
+    app.kubernetes.io/instance: controller-manager
+    app.kubernetes.io/component: manager
+    app.kubernetes.io/created-by: k0smotron
+    app.kubernetes.io/part-of: k0smotron
+    app.kubernetes.io/managed-by: kustomize
+spec:
+  selector:
+    matchLabels:
+      control-plane: controller-manager
+  replicas: 1
+  template:
+    metadata:
+      annotations:
+        kubectl.kubernetes.io/default-container: manager
+      labels:
+        control-plane: controller-manager
+    spec:
+      securityContext:
+        runAsNonRoot: true
+      containers:
+      - command:
+        - /manager
+        args:
+        - "--health-probe-bind-address=:8081"
+        - "--metrics-bind-address=127.0.0.1:8080"
+        - "--leader-elect"
```

Comment:
The current state of the patch file is:
```
        - "--health-probe-bind-address=:8081"
        - "--diagnostics-address=127.0.0.1:8080"
        - "--leader-elect"
        - "--enable-controller=bootstrap"
        - --feature-gates=${K0SMOTRON_FEATURE_GATES:=""}
```

And why we switching from patches to full files for controllers?